pkg/ast/spl/spl.peg

```diff
@@ -29,6 +29,15 @@ import (
     "github.com/siglens/siglens/pkg/segment/utils"
     log "github.com/sirupsen/logrus"
 )
+// ConvertCustomDateTimeFormatToEpochMs converts a date string in the format "MM/DD/YYYY:HH:MM:SS" 
+// to Unix time in milliseconds (epoch ms).
+func ConvertCustomDateTimeFormatToEpochMs(dateStr string) (int64, error) {
+    t, err := time.Parse("01/02/2006:15:04:05", dateStr)
+    if err != nil {
+        return 0, err
+    }
+    return t.UnixMilli(), nil
```

Comment:
Currently parser is assuming the time as UTC and just giving the epoch for that time. If I enter 10:00:00. It assumes that current UTC time is 10:00:00 and gives the output. You need to convert the time to UTC time and then return those epochs.